File: ocmsdk/src/main/java/com/gigigo/orchextra/core/domain/entities/article/ArticleHeaderElement.java
package com.gigigo.orchextra.core.domain.entities.article;

import com.gigigo.orchextra.core.domain.entities.article.base.ArticleElement;
import java.io.Serializable;

public class ArticleHeaderElement extends ArticleElement implements Serializable {

  private String html;
  private String imageUrl;
  private String imageThumb;

  public String getHtml() {
    return html;
  }

  public void setHtml(String html) {
    this.html = html;
  }

  public String getImageUrl() {
    return imageUrl;
  }

  public void setImageUrl(String imageUrl) {
    this.imageUrl = imageUrl;
  }

  public String getImageThumb() {
    return imageThumb;
  }

  public void setImageThumb(String imageThumb) {
    this.imageThumb = imageThumb;
  }
}

File: ocmsdk/src/main/java/com/gigigo/orchextra/core/domain/entities/article/ArticleHeaderElement.kt
package com.gigigo.orchextra.core.domain.entities.article

import com.gigigo.orchextra.core.domain.entities.article.base.ArticleElement
import com.gigigo.orchextra.core.domain.entities.article.base.ArticleElementRender

class ArticleHeaderElement : ArticleElement<ArticleHeaderElementRender>()

class ArticleHeaderElementRender : ArticleElementRender() {
  var html: String? = null
  var imageUrl: String? = null
  var imageThumb: String? = null
}
